Example/CoreDiagnostics/Tests/FIRCoreDiagnosticsTest.m

```diff
@@ -218,9 +220,7 @@ - (void)populateProto:(logs_proto_mobilesdk_ios_ICoreConfiguration *)config {
   config->dynamic_framework_count = numFrameworks;
   config->has_dynamic_framework_count = 1;
   config->apple_framework_version = FIREncodeString(combinedVersions);
-#if TARGET_OS_IOS
-  config->min_supported_ios_version = FIREncodeString(@"8.0");
-#else
+#if !TARGET_OS_IOS
```

Comment:
Why no min_supported_ios_version for iOS

Reply:
Just because in a test environment it fails to populate.